helm/freenas-provisioner/templates/secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.freenasConfig.secretName }}
```

Comment:
Should be relative to release name to be able to instanciate multiple times this chart.
I also like when I am able to pass an "existing secret" for a given chart and be able to tell the chart: "take this secret instead of the one included in the chart".